File: Assets/Backend/Entities/CharacterEntity.cs
using System;
using Unisave.Entities;
using UnityEngine;

public class CharacterEntity : Entity
{
    public EntityReference<AccountEntity> Account;
    public string Name;
    public string Gender;
    public string Race;
    public string Area;
    public Vector3 Position;
    public float Rotation;

    public int Level;
    public int TotalExp;

    public DateTime LastConnected;
    public DateTime LastDisconnected;
    
    public bool HasConnectedBefore;
}
[Serializable]
public class CharacterModel
{
    public string CharacterId;
    public string Name;
    public string Gender;
    public string Race;
    public string Area;
    public Vector3 Position;
    public float Rotation;

    public int Level;
    public int TotalExp;
}

File: node-backend/libs/character/src/entities/character.entity.ts
import { Column, Entity, PrimaryColumn, PrimaryGeneratedColumn, Unique } from 'typeorm'

@Unique('name', ['name'])
@Entity('character')
export class CharacterEntity {
  @Column('varchar')
  accountId: string
  @PrimaryColumn('varchar')
  name: string
  @Column('varchar')
  gender: string = 'male'
  @Column('varchar')
  race: string = 'human'
  @Column('varchar')
  area: string = 'Rein'
  @Column('decimal')
  position_x: number = 0
  @Column('decimal')
  position_y: number = 0
  @Column('decimal')
  position_z: number = 0
  @Column('decimal')
  rotation: number = 0

  @Column('int')
  level: number = 1
  @Column('bigint')
  exp: number = 0

  @Column('bigint')
  lastConnected: number = 0
  @Column('bigint')
  lastDisconnected: number = 0

  @Column('tinyint')
  hasConnectedBefore: boolean = false
}